### PR TITLE
Nigeriobike

### PR DESCRIPTION
**Pull request by Nigeriobike**

### CODE REVIEW

File: src/test/java/holders/TestInit.java [hunk 1]
```diff
@@ -18,7 +19,7 @@ public class TestInit {
     public WebDriver driver;
     public Actions action ;
 
-    @BeforeClass
+    @BeforeMethod
```
Comment (DereviankinNikita) [inline src/test/java/holders/TestInit.java:22]: BeforeMethod?

File: testng.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (DereviankinNikita) [inline testng.xml]: Create separate directory for suites

**Review by DereviankinNikita — CHANGES_REQUESTED** [review]

File: src/test/java/holders/TestInit.java (hunk 1 above)
Comment (Nigeriobike) [reply src/test/java/holders/TestInit.java:22]: Yes , cos I use parameters for my test function in classes

File: testng.xml (hunk 2 above)
Comment (Nigeriobike) [reply testng.xml]: I can't cos of data provider

**Comment by DereviankinNikita** [comment]
@Nigeriobike please fix merge conflicts

**Comment by Nigeriobike** [comment]
Fixed

**Review by DereviankinNikita — APPROVED** [review]

**Review by CyberShval — APPROVED** [review]

**Pull request merged**